oatomobile/baselines/torch/dim/model.py

```diff
@@ -31,6 +31,7 @@
 from oatomobile.torch.networks.mlp import MLP
 from oatomobile.torch.networks.perception import MobileNetV2
 from oatomobile.torch.networks.sequence import AutoregressiveFlow
+from oatomobile.torch.networks.tcn import TCN
```

Comment:
add `TCN` in `oatomobile.torch.networks.sequence`

Reply:
Resolved.